### ISSUE TITLE
Guidelines needed for heating/cooling measures that are not 1-1 replacements

### ISSUE THREAD

**Issue by GamalielL** [hpxmlwg/hpxml#84]
I think it would be helpful to establish guidelines for how to represent components in complex heating/cooling system measures.  Here are some examples and questions

**Scenario 1**
Existing systems:
-Propane furnace, 100% of load
Proposed systems:
-Old propane furnace, 60% of load
-New wood Stove, 40% of load
Question: Should this be represented as one or two measures:
Thoughts: A single measure seems to be clearly the best choice to capture the cost effectiveness of adding the new system, but it's more complicated.
Question: Should the measure reference the propane furnace as a replaced/installed component?
Thoughts: Including it is helpful for understanding the full energy impact of the measure, but may cause problems for automated incentive parsing.  It's problematic that there is no convention for identifying that the furnace described in the proposed building is the old furnace, but for the change in load. 
Question: Should the savings represent the total change in usage of both fuels?
Thoughts: The seems to be an easy "yes" for this scenario, but it's not so clear when more then two systems are involved in load shifting.

**Scenario 2**
Existing systems:
-Gas furnace , 100% of heating load
-Central AC, 100% of cooling load
Proposed systems:
-New Central Heat pump, 100% of both heating and cooling

Question: Should this be represented as a single combined measure or separate heating and cooling measures?
Thoughts: This seems like and easy "yes", as combining them allow for easier evaluation of the cost effectiveness of the new system, but there may be stakeholder who want the heating and cooling evaluated separately and the basic use case does not require inclusion of end use savings in the measure

**Scenario 3**
Existing systems:
-Gas furnace , 100% of heating load
-Central AC, 100% of cooling load
Proposed systems:
-New Gas furnace , 50% of heating load
-Central Heat pump, 50% of heating and  100% of cooling

Question:  Is there a reasonable way to put the new furnace and heat pump in separate measures?  Would you assign the old furnace as a replaced component in both measures?  How should we allocate savings? 
Thoughts: My preferred solution is to treat this as a single measure and avoid all of these questions, but this may be problematic for incentive processing.  

**Scenario 4**

Existing systems
-Propane furnace 70 AFUE, 70% of load
-Electric baseboard, 30% of load
Proposed systems
-New propane furnace 90 AFU, 60% of load
-New wood Stove, 30% of load
-Old electric baseboard, 10% of load

This doesn't really add a new question, but it show how complex things can get.  I've seen jobs like this.  Replace the wood stove with a mini-split and then you have cooling to deal with a well.  I think this supports the argument for just combining everything involved in load shifts.  Developing a set of guidelines that would clearly define how to split situations like this into multiple measures may be difficult.

**Comment by brandongallagher** [comment]
I agree guidelines would be helpful.  I pretty much agree that all of those scenarios should be represented as a single measure, but I’m also not really sure how it will affect the incentives.  My quick answer if it ever does need to be separate measures is to split the hvacs up in to pseudo systems that can be entirely replaced/installed by the measure.  For example, in Scenario 3 model the existing furnace as two systems, each at 50%, then replace one with the new furnace and the other (along with the A/C) with the heatpump.

 

-brandon

**Comment by GamalielL** [comment]
@brandongallagher 
Your idea does a good job of communicating how the savings would be calculated, but it would be unfortunate to lose the ability to accurately convey the systems in the existing building.  I would prefer that we come up with a convention that doesn't disrupt accurate building descriptions.  It would also be tricky to build a generator that created those extra systems when needed. If a program needed to have scenario 3 represented as two measures, they could always instruct their users to split the furnace in the model.

**Comment by Karchie1** [comment]
We (the HPwES program team) met to discuss these scenarios in New York.  Here are our responses:

Scenario 1
We would want a single measure that includes only the wood stove as an installed component, pointing to the furnace as a replaced component.  We would want to see the “energysavingsinfo” tree populated with savings that correctly show negative wood fuel savings equal to the 40% of the load that is being offset be the new stove, and positive propane savings in line with the 40% load shaving of the furnace.  If we don’t see an installed furnace, we can deduce that the pre and post furnaces are the same.  We would not want to see the old furnace listed anywhere as an installed component because it would likely not pass our eligibility criteria and would be confusing from an evaluation perspective.  We need to be clear that no new furnace has been installed in this scenario.

Scenario 2
One measure with net savings for heated and cooling by fuel type.  There are end use savings by fuel type already in HPXML.  If a program needs to know heat vs cooling electric savings, they should require / use those elements.

Scenario 3
There are two ways of handling this, as Gamaliel suggests.  One would be to handle is as a single measure with the old furnace as the replaced component, and the new furnace and heat pump being the installed components.  Savings would net out.  This works in principle, but it may be tricky for our portal to parse out.  Having different types of systems (i.e. heat pumps vs furnaces) referenced in the same measure would require that each scenario that has this type of disparity be accounted for and the logic written to handle it. Plus, in our program, furnaces are incentivized, but heat pumps are not.  Bundling the costs together into one measure would make it very difficult to properly allocate incentives.  Lastly, if one of the two system replacements resulted in net negative dollar savings (like if we were going oil to propane), there would be no way to know that, and our program requires all measures have positive savings to qualify.  Therefore, this option would present multiple problems for our program.
This leads to the second option, which would be to have one measure for the new furnace and one for the heat pump.  Both would point to the old furnace as the replaced component, and the savings would be split proportional to the load split.  The logic to handle this is already in place.  

Scenario 4
This is a mix of scenarios 1 and 3.  We would want to handle this in 2 measures; one for the furnace and one for the stove.  Both the furnace and stove would point to the old furnace and the baseboard as the replaced components, with savings allocated proportional to the delta in the loads.  For instance, the wood stove would be offsetting 10% of the propane and 20% of the electric heating consumption.  With fuel neutrality, it might be possible to do something simpler, like having 2/3 of both the electric and propane savings assigned to the furnace and 1/3 assigned to the stove, with the negative wood savings split in a similar fashion, but that would likely cause problems for programs where funding is tied to fuels.  Even in NY, this would still likely be problematic because some of the customers may live in non-contributing electric territories.  Switching from electricity to another fuel becomes complicated for those customers’ projects.

**Comment by GamalielL** [comment]
@Karchie1 
OptiMiser currently supports your vision for Scenarios 1&2 with the exception that we do not include systems that only have a reduction in load assignment as a replaced component.  How important is that to your processing?
Supporting your preferences for 3&4 would be a significant lift for us and not something we would want to undertake unless it was following a guideline agreed upon by the HPXML community.  Currently we report scenarios using a single measure.  I don't expect that we will see a lot of jobs with multiple new systems, so we can afford to take time with this.

**Comment by GamalielL** [comment]
Below is proposed logic for calculating separate savings for every installed system.  

- If an existing system is removed with no designated replacement, then it’s consumption goes into a pool of displaced consumption.
- If an existing system is left in place, but the load fraction that it serves is reduced, then the amount by which its consumption is reduced is added into the pool of displaced consumption.
- If an existing system is replaced by new system meeting the same load fraction then savings of the new system is a simple difference in consumptions.
- If an existing system with load fraction F1 and consumption C1 is replaced by a new system meeting a SMALLER load faction F2 generating consumption C2, then the savings of the new system savings is computed as C1*F2/F1-C2.  The excess consumption C1(1-F2/F1) is goes into the pool of displaced consumption.
- If an existing system with load fraction F1 and consumption C1 is replaced by a new system meeting a LARGER load faction F2 generating consumption C2, then the savings of the new system savings is computed as C1+Cd*(F2-F1)/Fd-C2, where Cd is the total displaced consumption pool and Fd is the load faction represented by the displaced consumption.
- If a new system with load fraction F2 and consumption C2 is installed and is not designated as a replacement of any existing system, then the savings of the new system savings is computed as Cd*F2/Fd, where Cd is the total displaced consumption pool and Fd is the load faction represented by the displaced consumption.

**Comment by GamalielL** [comment]
Attaching several sample HPMXL files for complicated systems measures.  In these files I use a separate measure for each new system.  Removed systems are only identified if they are directly identified as replaced by an installed system (by the user) or the only change to systems of that type (e.g heating systems) was the removal of an existing system with no new system installed. 
[SampleSystemsMeasures.zip](https://github.com/hpxmlwg/hpxml/files/551677/SampleSystemsMeasures.zip)
